src/rabbit_auth_backend_ldap.erl

```diff
@@ -30,6 +30,7 @@
 -define(L(F, A),  log("LDAP "         ++ F, A)).
 -define(L1(F, A), log("    LDAP "     ++ F, A)).
 -define(L2(F, A), log("        LDAP " ++ F, A)).
+-define(HIDDEN_CRED,  "xxxx").
```

Comment:
"Hidden" makes me think that we are doing something shady. Can this be renamed to "SCRUBBED"? And lets expand the "cred" while we are at it.